pkg/planner/core/casetest/cascades/testdata/cascades_suite_out.json

```diff
@@ -0,0 +1,107 @@
+[
+  {
+    "Name": "TestDeriveStats",
+    "Cases": [
+      {
+        "SQL": "select count(1) from t1 group by a, b",
+        "Str": "GID:1, logic prop:{stats:{count 5, ColNDVs map[1:2 2:2]}, schema:{Column: [test.t1.a,test.t1.b] Key: [] Unique key: []}}\nGID:2, logic prop:{stats:{count 4, ColNDVs map[4:4]}, schema:{Column: [Column#4] Key: [] Unique key: []}}\nGID:3, logic prop:{stats:{count 4, ColNDVs map[4:4]}, schema:{Column: [Column#4] Key: [] Unique key: []}}"
```

Comment:
The `\n` seems to be parsed in the wrong way?

Reply:
json string seems couldn't switch line, i change it to string slice, which will be much more clear
![image](https://github.com/user-attachments/assets/e3ab5f49-27ae-4aab-870b-9556e6687417)